### PR TITLE
Added replaceAll parameter to update endpoint

### PR DESCRIPTION
**Pull request by dave1010**
ChatGPT did most of the work here, editing the plugin itself.

It updated the JS fine with the existing `update` command.

It then tried to use the replaceAll param and got back `UnrecognizedKwargsError`. That seems to be an OpenAI thing. ChatGPT did some testing and it happened whenever it called an API endpoint with a param that's not in the openapi.yaml. Presumably that's a safety thing.

We went round in circles a bit here, thinking it was some kind of next.js cache! Then we realised we could make changes to the messages immediately.

The solution is updating the openapi.yaml file, then clicking "Refresh plugin" in the ChatGPT Plugin devtools sidebar (for anyone else reading this, you get to that from the ChatGPT settings on the bottom left.)

Once I'd done this, `replaceAll` worked fine and ChatGPT did some tests to check the functionality works fine.

### CODE REVIEW

File: public/openapi.yaml [hunk 1]
```diff
@@ -147,6 +147,9 @@ paths:
                 replacementString:
                   type: string
                   description: The string to replace the search string with
+                replaceAll:
+                  type: boolean
+                  description: Whether to replace all occurrences of the search string. Optional.y
```
Comment (ykdojo) [inline public/openapi.yaml:152]: Optional.y - typo?

**Comment by ykdojo** [comment]
Thank you for this PR! I added a small comment.

**Comment by ykdojo** [comment]
@dave1010 why did you close? Sorry I didn't see your earlier commit

**Comment by dave1010** [comment]
Oops, sorry. I was cleaning up branches and projects and accidentally removed this one. I've just submitted the same PR again, as I don't think I can reopen this one.

https://github.com/ykdojo/kaguya/pull/28

**Comment by ykdojo** [comment]
No worries and thank you!

**Pull request closed without merging**